core/sagas/inputs.js

```diff
@@ -73,23 +75,68 @@ function flattenInput(response) {
       release: item.builds[0].release,
       arch: [item.builds[0].arch]
     };
-    if (previousBuilds.hasOwnProperty(build.version + build.release)) {
-      // if this item has the same values for release and version of a
-      // previousBuild
-      // then push arch to the array of arch's, and filter this item out
-      previousBuild = previousBuilds[build.version + build.release];
+    if (previousBuilds.hasOwnProperty(build.version)) {
+      // if this item has the same version value as a
+      // previousBuild replace the release value (this is assumed to be the latest)
+      // then push arch to the array of arch's and filter this item out
+      previousBuild = previousBuilds[build.version];
       previousBuild.arch = previousBuild.arch.concat(build.arch);
+      previousBuild.release = build.release;
       return false;
     } else {
       // else push the build to the array of builds
       flattened.builds = [build].concat(flattened.builds);
     }
-    previousBuilds[build.version + build.release] = build;
+    previousBuilds[build.version] = build;
     return true;
   });
+  flattened.builds = addWildcardVersions(flattened.builds);
   return flattened;
 }
 
+function addWildcardVersions(builds) {
+  // create a new array of builds that includes wildcard versions
+  let wildcardBuilds = [];
+  builds.forEach(item => {
+    item.depsolveVersion = item.version;
```

Comment:
Is `depsolveVersion` ever different from `version`? If it is, why do you overwrite it here?

Reply:
Yes, the version could be "10.*" but the depsolveVersion is "10.11.0". Keeping the depsolveVersion associated with the wildcard versions allow me to show what the current depsolved version of "10.*" would be.